VideoWeb/VideoWeb/ClientApp/src/app/services/events.service.ts

```diff
@@ -109,10 +109,10 @@ export class EventsService {
 
     this.connection.on(
       'ReceiveMessage',
-      (conferenceId: string, from: string, message: string, timestamp: Date, messageUuid: string) => {
+      (conferenceId: string, from: string, message: string, timestamp: Date, messageUuid: string, isJudge: boolean) => {
```

Comment:
No neeed to extend the model here, not a suitable solution for when participant IM is enabled.

Use the from property

Reply:
done